facia-tool/public/package.json

```diff
@@ -8,17 +8,17 @@
     "configFile": "js/jspm-config.js",
     "dependencies": {
       "EventEmitter": "npm:wolfy87-eventemitter@4.2.11",
-      "css": "^0.1.0",
-      "font-awesome": "4.2.0",
-      "jquery": "2.1.3",
+      "css": "github:systemjs/plugin-css@^0.1.0",
+      "font-awesome": "npm:font-awesome@4.2.0",
+      "jquery": "github:components/jquery@2.1.3",
```

Comment:
I suspect that it's a change in JSPM. I've seen it in facia-tool as well after installing JSPM 0.15. They used to have their own registry but apparently now they simply alias to the either npm or github.